### DISCUSSION TITLE
requesting some common keyboard shortcuts

### DISCUSSION THREAD

**Discussion by Norgus** [plateaukao/einkbro#292] [Ideas]
It would be nice to have the common shortcuts available in many (most?) browsers and platforms. I quite often use my eink device with a bluetooth keyboard, and I see there is recognition of this use-case with optional vi key bindings.

The one's that come to mind are:
* ctrl + t = new tab
* ctrl + w = close tab
* ctrl + tab = next tab
* ctrl + shift + tab = previous tab
* ctrl + number = jump to that number tab in the list
* ctrl + f = open the in-page find textbox in focus
* ctrl + e = edit the address of the current tab

Maybe some common F-key functions like reload and fullscreen, but I think keyboards are often treated as having special android buttons in their place, so it might cause more hastle than it's worth with confused users.

I'd like to add that your app makes browsing so much better than the experience with other browsers, and I really appreciate it. Thanks for developing and improving/maintaining it, awesome work!

**Comment by gety9** [comment]
@Norgus  

!!!Strongly!!! upvoting idea as follow hard keyboard user (i opened similar discussion sometime ago https://github.com/plateaukao/einkbro/issues/73)

If Daniel chooses to imlement this functionality maybe it will be easier to simply allow us to rebind exisitng vi keys (from their vi equivalent to say ctrl-f, etc) than adding something new. At least it will be a start :)

But i agree, the lack of ctrl + KEY shortcuts is pretty much only thing Eink Bro lacks (comparing to Chrome, Brave, etc). Other than that it's amazing browser.

**Comment by alchem1no** [comment]
@Norgus 

Good idea! I'm planning to buy Bluetooth keyboard in the future for the phone and pretty much any other device in general. 

I think it would also be great to make an option for setting custom keybindings like Ctrl + I for Incognito mode, for example.

**Comment by plateaukao** [comment]
pull request is welcomed.
